website/package.json

```diff
@@ -23,7 +23,7 @@
     "@docusaurus/plugin-pwa": "2.0.0-alpha.dbfa256a7",
     "@docusaurus/preset-classic": "2.0.0-alpha.dbfa256a7",
     "clsx": "^1.1.1",
-    "docusaurus-plugin-sass": "^0.1.11",
+    "docusaurus-plugin-sass": "git+https://github.com/slorber/docusaurus-plugin-sass.git#patch-2",
```

Comment:
Perfect candidate for the `patch:` protocol which is much faster than git installs - https://yarnpkg.com/features/protocols#patch

Reply:
patch over git dependency would be awesome 👍 we already patch a few packages, so would also fit in with our current approach

Reply:
TIL didn't know Yarn 2 had built-in patch-package! cool

Updated to:

```
    "docusaurus-plugin-sass": "patch:docusaurus-plugin-sass@^0.1.11#./patches/docusaurus-plugin-sass.diff",
```